smokes/e2e/about.scenarios.coffee

```diff
@@ -3,14 +3,26 @@
 # to use previous and next link
 
 aboutPage = require('./pages/about.coffee')
+homePage = require('./pages/home.coffee')
+builderPage = require('./pages/builder.coffee')
 
-describe('', () ->
+describe 'about test', () ->
     about = null
+    home =  null
+    builder = null
 
-    beforeEach(() ->
+    beforeEach () ->
         about = new aboutPage('runtests')
-    )
+        builder = new builderPage('runtests', 'force')
+        home = new homePage()
+        builder.goDefault()
```

Comment:
tests done without builder.goDefault or with browser.sleep(500) go in failure with the following error message: 
"Failed: Error while waiting for Protractor to sync with the page: "window.angular is undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor's bootstrapping.  See http://git.io/v4gXM for details"

I will to keep it temporary. We can discuss about it